responses/content.js

```diff
@@ -37,28 +37,47 @@ function inputWasNotUnderstood(isRepeating, input = null, options = null, seedPe
 }
 
 function theAnswerGivenWasCorrect(articleData, newQuestion, people){
-
-	const illustration = (articleData.mainImageUrl !== null)?articleData.mainImageUrl:process.env.FT_LOGO;
-
-	return {
-		displayText : `Correct! ${people.submitted.replace('people:', '')} was mentioned with ${people.seed.replace('people:', '')} in the FT article:`,
-		speech : `Correct! ${people.submitted.replace('people:', '')} was mentioned with ${people.seed.replace('people:', '')} in the FT article titled: ${articleData.title}.`,
-		ssml : `<speak>Correct! ${people.submitted.replace('people:', '')} was mentioned with ${people.seed.replace('people:', '')} in the FT article titled: ${articleData.title}. <break time="1s"/></speak>`,
-		article: articleData.title,
-		link: `https://ft.com/${articleData.id}`,
-		image: illustration,
-		question: newQuestion,
-		chips: newQuestion.chips
+	// it is possible (albeit bad) that articleData might be null, so handle that situation
+
+	let responseObj = {
```

Comment:
`responseObj` can be a const, as you're only changing properties, not the object itself.